docs/src/main/sphinx/sql/explain.rst

```diff
@@ -615,10 +620,37 @@ Validate::
     -------
      true
 
+If the statement is not correct because syntax error such as an unknown keyword
+are found, the error message details the problem::
```

Comment:
```suggestion
If the statement is not correct because a syntax error, such as an unknown keyword,
is found, the error message details the problem::
```